apps.yml

```diff
@@ -602,18 +602,6 @@ apps:
     url: https://developer.mozilla.org/
     vanity_url:
     - /mdn
-- application:
-    authorized_groups:
-    - everyone
-    authorized_users: []
-    client_id: nPr8QRo0dLxM3RRHwIXRSvhmOSPDvNr4
-    display: true
```

Comment:
display:true and logo:moderator.png are going away; the surviving entry is display:false and logo:auth0.png.  Should these transfer over too?